Makefile

```
@@ -43,13 +44,22 @@ docs/volto:
 	git submodule update; \
 	ln -s ../submodules/volto/docs/source ./docs/volto
 
-docs/restapi:
+docs/plone.restapi:
 	git submodule init; \
 	git submodule update; \
 	ln -s "../submodules/plone.restapi" "./docs/plone.restapi"
```

The quotes that I added are not necessary. It is an artifact of my failed tests to fix this target.

```suggestion
	ln -s ../submodules/plone.restapi ./docs/plone.restapi
```